site/source/_posts/2018/03/11/Dojo-2-rc1.md

```diff
@@ -0,0 +1,186 @@
+
+`@dojo/widget-core` embraces the future of the web platform with out of the box meta implementations for Intersection Observers, Web Animations, and other APIs, with plans shortly for additional emerging standards such as ResizeObserver.
+
+An example of a Dojo 2 widget using the Intersection Observer meta is:
+
+```typescript
+import { WidgetBase } from '@dojo/widget-core/WidgetBase';
+import { v, w } from '@dojo/widget-core/d';
+import { Intersection } from '@dojo/widget-core/meta/Intersection';
+
+interface ImageProperties {
+	image: string;
+}
+
+class Image extends WidgetBase<ImageProperties> {
+	protected render() {
+		const { image } = this.properties;
+		const { isIntersecting } = this.meta(Intersection).get('root');
+		const properties = isIntersecting ? { key: 'root', src: image } : { key: 'root' };
+
+		return v('img', properties);
+    }
+}
+
+interface ImageListProperties {
+	images: string[];
+}
+
+export class ImageList extends WidgetBase<ImageListProperties> {
+	protected render() {
+		const { images } = this.properties;
+			return v('div', images.map((image) => {
+				return v('ul', { key: image }, [ w(Image, { image } )]);
+		}));
+	}
+}
+```
+
+Note that if you prefer React's JSX syntax, you can replace Hyperscript with TSX in the `render` function, e.g.:
+
+```typescript
+protected render() {
+	const { images } = this.properties;
+	const items = images.map((image) => (
+		<ul key={image}>
+			<Image, image={image}/>
+		</ul>
+	));
+	return <div>{items}</div>;
+}
+```
+
+## Dojo loves Custom Elements
+
+A top priority for Dojo is interoperability with the web platform, both by leveraging current and emerging standards and providing a mechanism to coexist with other frameworks efficiently. The Custom Elements portion of the Web Components standard is one area to encourage interoperability between UI component libraries and frameworks.
+
+The [Custom Elements Everywhere](https://custom-elements-everywhere.com/) project was released last year to highlight framework support for custom elements, and Dojo currently has a perfect score. Within Dojo 2, you may use custom elements, express Dojo 2 widgets as custom elements, and easily export Dojo 2 widgets as custom elements via one of our `@dojo/cli` commands, making it easy to share and distribute custom elements for easy use across projects and frameworks! So if you're in an organization where some teams use Angular, some use React, some use Vue, and some use Dojo, we provide a future today where you can write a component once and share it across those teams!
+
+By default, Dojo's custom elements are built for evergreen browsers to reduce overall bundle size, meaning that most basic widgets are <20KB (gzipped) which includes all of the @dojo runtime needed to support the custom element.
+
+To create or use a Dojo 2 web component,
+
+```
+<div>
```

Comment:
Shall we show the actual helloworld widget with the `customElement` decorator? Not sure the html below adds anything...